File: pyramid_block.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Get height
    int height;
    do
    {
        height  = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    //printf("Stored: %i\n", height);

    for (int i = 0; i < height; i++)
    {
        //moving towards right
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        //building left pyramid
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        printf("  ");

         //building right pyramid
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: CS50/pyramid_block.py
from cs50 import get_int

# check for positive integer less than 9
while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break

for i in range(height):
    # moving to the right
    for j in range(height - i - 1):
        print(" ", end="")

    # building left block
    for j in range(i + 1):
        print("#", end="")

    # space
    print("  ", end="")

    # building right block
    for j in range(i + 1):
        print("#", end="")

    print()
